deeprankcore/query.py

```diff
@@ -645,13 +683,14 @@ def _load_ppi_atoms(pdb_path: str,
     return contact_atoms
 
 
-def _load_ppi_pssms(pssm_paths: Union[Dict[str, str], None],
-                    chain_id1: str, chain_id2: str,
+def _load_ppi_pssms(pssm_paths: Optional[Dict[str, str]],
+                    chains: List[str],
```

Comment:
Right now we don't support more than 2 chains for PPI, so I think it's better to have chain_id1 and chain_id2 to make it more explicit

Reply:
I made the change mainly because the linter was complaining, but also think it's nicer because a few lines below we anyway loop through the two chains turn them into a list there. 
I think because it's an internal function, it doesn't matter too much how explicit it is now. And your comment makes me realize that if in the future we would want to allow more than 2 chains in a PPI, then this is one less issue to think about :)